tests/unit/button/button_methods.js

```diff
@@ -28,47 +28,11 @@ test( "refresh: Ensure disabled state is preserved correctly.", function() {
 	element.button( { disabled: true} ).button( "refresh" );
 	ok( element.button( "option", "disabled" ), "<button> should remain disabled after refresh");
 
-	element = $( "<input type='checkbox'>" );
-	element.button( { disabled: true} ).button( "refresh" );
-	ok( element.button( "option", "disabled" ), "Checkboxes should remain disabled after refresh");
-
-	element = $( "<input type='radio'>" );
-	element.button( { disabled: true} ).button( "refresh" );
-	ok( element.button( "option", "disabled" ), "Radio buttons should remain disabled after refresh");
 
 	element = $( "<button></button>" );
 	element.button( { disabled: true} ).prop( "disabled", false ).button( "refresh" );
 	ok( !element.button( "option", "disabled" ), "Changing a <button>'s disabled property should update the state after refresh."); //See #8828
```

Comment:
Let's move inline comments where they belong, or drop them entirely. Even better would be a test that iterates through all options and verifies that `refresh` doesn't mess up any of them.